iot-hub/Samples/device/ConventionBasedOperations/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -15,6 +15,22 @@ namespace Microsoft.Azure.Devices.Client.Samples
 {
     public class TemperatureControllerSample
     {
+        // The specific response codes in this sample.
+        public class ClientResponseCodes : CommonClientResponseCodes

```

Comment:
nit: I'd separate this out into its own class file so that it is easily discoverable.